setup_base.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# This script installs a Jamulus repository to debian based systems
+
+REPO_FILE=/etc/apt/sources.list.d/jamulus-~§{R_NAME}§~.list
+KEY_FILE=/etc/apt/trusted.gpg.d/jamulus-~§{R_NAME}§~.asc
+
+echo "Setting up Jamulus repo at ${REPO_FILE}..."
+echo 'deb https://github.com/~§{R_GITHUB_REPOSITORY}§~/releases/download/~§{R_NAME}§~ ./' > ${REPO_FILE}
```

Comment:
~§ can be replaced with anything else.

Reply:
I guess we could use `envsubst` here as well?

Besides that, why woud you move it to a more prominent file in the first place? I fear it might trigger users to download it as-is which won't work. Maybe some `.template` suffix would help to avoid that issue.

Do you like the separate file more as it allows dropping the heredoc?

Reply:
> Do you like the separate file more as it allows dropping the heredoc?

Exactly. I don't really like a bash script embedded in a bash script (feels strange)

Reply:
> I guess we could use envsubst here as well?

Didn't know about this command till now. We'd need to `export` the respective variables then?